bin/get-docs.sh

```diff
@@ -5,22 +5,22 @@ set -e
 pwd=$PWD
 
 docs=$pwd/docs
-rm -rf $docs
-mkdir -p $docs
+rm -rf "$docs"
+mkdir -p "$docs"
 
 deps=$pwd/deps
 
-cd $deps/renovate
+cd "$deps/renovate"
 
-if [ -z $SKIP_BUILD ]; then
+if [ -z "$SKIP_BUILD" ]; then
   yarn install --frozen-lockfile
   yarn build:docs
 fi
 
-cp -R tmp/docs/* $docs
+cp -R tmp/docs/* "$docs"
 
-cd $pwd
+cd "$pwd"
 cp -R src/* docs/
 
-echo '---\ntitle: Merge Confidence\ndescription: Node versions support in Renovate\n---\n' > docs/merge-confidence.md
+printf '---\ntitle: Merge Confidence\ndescription: Node versions support in Renovate\n---\n' > docs/merge-confidence.md
```

Comment:
`printf` is failing, we're not giving the input in the way it expects.

I don't know how to change the script so that `printf` will print the correct string into the `docs/merge-confidence.md` file.

Reply:
```suggestion
printf -- '---\ntitle: Merge Confidence\ndescription: Node versions support in Renovate\n---\n\n' > docs/merge-confidence.md
```